src/Lean/Server/Watchdog.lean

```diff
@@ -210,6 +224,10 @@ section ServerM
           fw.erasePendingRequest id
         if let Message.responseError id _ _ _ := msg then
           fw.erasePendingRequest id
+        if let Message.notification "$/lean/ileanInfo" params := msg then
+          if let some params := params then
+            if let Except.ok params := FromJson.fromJson? $ ToJson.toJson params then
+              handleIleanInfo fw params
```

Comment:
I'm late to the party, but do we actually want to forward `$/lean/ileaninfo` to the client? Maybe there should be a `return` here. @Garmelon

Reply:
True, but would a `return ← loop` from inside the `try` lead to some sort of leak because of all the nested `try`s? I guess the check for `$/lean/ileanInfo` can just be moved to the top.